birdie_snapshots/codegen_discriminator_as_elements_test.accepted

```diff
@@ -88,11 +88,11 @@ pub fn data_decoder() -> decode.Decoder(List(Data)) {
 
 pub fn data_to_json(data: List(Data)) -> json.Json {
   json.array(data, fn(data) { case data {
-    Up(height:) -> json.object([
+    DataUp(height:) -> json.object([
```

Comment:
Ah! I think my bug is still here. The type definition above in this file says `DataElementUp`, but this is `DataUp`

Reply:
I might actually have fixed this locally already, let me just push my recent changes.